src/Container.php

```diff
@@ -89,7 +89,6 @@
 use Infection\TestFramework\PhpUnit\Config\XmlConfigurationHelper;
 use Infection\TestFramework\PhpUnit\Coverage\IndexXmlCoverageParser;
 use InvalidArgumentException;
-use function is_callable;
```

Comment:
Why removing this one? Isn't it used in `offsetAdd()`?

Reply:
Nope this is no longer needed